packages/calypso-e2e/src/lib/pages/editor-page.ts

```diff
@@ -133,18 +134,20 @@ export class EditorPage {
 	 *
 	 * @returns {Promise<Frame>} iframe holding the editor.
 	 */
-	async getEditorFrame(): Promise< Frame > {
-		const locator = this.page.locator( selectors.editorFrame );
-
-		const elementHandle = await locator.elementHandle( {
-			timeout: 105 * 1000,
-		} );
+	async getEditorFrame(): Promise< Frame | Page > {
+		// Return the page object as Atomic editor permits direct
+		// access.
+		if ( this.target === 'atomic' ) {
+			return this.page;
+		}
```

Comment:
For Atomic non-framed editor, the editor itself is accessible straight from the page, thus the page object itself is returned.

As per the Playwright documentation, `Page` and `Frame` both support the same methods. This is because `Page.methodCall` is merely a shortcut for the primary frame's `Frame.methodCall`. 

Name of this method `getEditorFrame` is up for debate, as it is no longer only returning a frame. Perhaps `getEditorHandle`, or `getEditor`?

Reply:
I like `getEditorHandle`!

This may be way out of scope for this PR, but now that we have this private `editor` property, could we just ditch the `getEditorFrame` function entirely and start using that property instead? We'd have to update the syntax of the other methods, but I think it might simplify things a good bit!

Reply:
I think `getEditorHandle` is a good name 👍🏻

Reply:
> This may be way out of scope for this PR, but now that we have this private editor property,
Ah, that'd be even better.

Reply:
I looked at simply returning `this.editor`, but one reason I was prevented from doing this is that for some methods (eg. `dismissWelcomeTour`) the frame handle is still required as `waitForFunction` only exists for Page/Frame objects.

Reply:
Still, worth looking into as part of followup, if so desired.

Reply:
Yeah, that seems like maybe too much scope creep for this PR. That sounds good to me!